coredhcp.go

```diff
@@ -136,6 +141,16 @@ func (s *Server) MainHandler6(conn net.PacketConn, peer net.Addr, req dhcpv6.DHC
 		}
 	}
 
+	// if the request was relayed, re-encapsulate the response
+	if req.IsRelay() {
+		tmp, err := dhcpv6.NewRelayReplFromRelayForw(req.(*dhcpv6.RelayMessage), resp.(*dhcpv6.Message))
```

Comment:
The 2nd type assertion is no longer needed.

Reply:
`resp` is still of `dhcpv6.DHCPv6` type, the assertion is still needed unfortunately :( (the compiler complains otherwise)

Reply:
Oh right, my bad.